### DISCUSSION TITLE
Docker on Raspberry PI

### DISCUSSION THREAD

**Discussion by kylow-may** [thirtythreeforty/neolink#296] [General]
Is there a docker version for Raspberry PI?  The current docker version is not designed to run on raspberry.

**Comment by QuantumEntangledAndy** [comment]
I believe my last built one was quantumentangledandy/neolink:three-stage-docker it's getting a bit old now though. It's not hard to make your own image. We have binaries for armv7 so you don't even have to compile just copy into the image.
